draft-js-plugins-editor/src/Editor/index.js

```diff
@@ -1,22 +1,21 @@
 /* eslint-disable no-continue,no-restricted-syntax */
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import {
-  Editor,
   EditorState,
+  Editor,
   DefaultDraftBlockRenderMap,
 } from 'draft-js';
-import { List, Map } from 'immutable';
-import MultiDecorator from './MultiDecorator';
-import createCompositeDecorator from './createCompositeDecorator';
-import moveSelectionToEnd from './moveSelectionToEnd';
+import { Map } from 'immutable';
 import proxies from './proxies';
+import moveSelectionToEnd from './moveSelectionToEnd';
+import resolveDecorators from './resolveDecorators';
 import * as defaultKeyBindingPlugin from './defaultKeyBindingPlugin';
 
 /**
  * The main editor component
  */
-class PluginEditor extends Component {
+class PluginEditor extends PureComponent {
```

Comment:
Please don't make it a PureComponent: They can't have `ref`s, so you can't do `this.editor.focus()` and stuff like that.

Testing the patch now!